src/test_report.py

```diff
@@ -20,6 +26,34 @@ def __init__(self, configs, args):
         self._db_config = configs['db_configs'][args.db_config]
         api_token = os.getenv('API_TOKEN')
         self._db = kernelci.data.get_db(self._db_config, api_token)
+        self._email_hostname = args.email_hostname
+        self._email_hostport = args.email_hostport
+
+    def create_email(self, template):
```

Comment:
Name `template` may be misleading. What is passed to the function is rather a content which is rendered from the template and passed to the function.

Reply:
updated

Reply:
Erm, I still can se the variable named `template`